### PR TITLE
[1907] Update bullet point re allocations in rollover

### PR DESCRIPTION
**Pull request by AbigailMcP**
### Context

https://trello.com/c/LJWi1K8l/1907-s-allocations-bullet-point-should-be-in-both-cycles

### Changes proposed in this pull request

- Updates the content for the allocations bullet point seen during rollover
- Copies the bullet point into the 'Next cycle' section

### Guidance to review

This is what it now looks like 'During rollover':

Screenshot 2021-06-21 at 10 39 30

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally
- [ ] Product Review

### CODE REVIEW

**Comment by AbigailMcP** [comment]
[[S] Allocations bullet point should be in both cycles](https://trello.com/c/LJWi1K8l/1907-s-allocations-bullet-point-should-be-in-both-cycles)

**Review by dankmitchell — DISMISSED** [review]

File: app/views/providers/_show_during_rollover.html.erb [hunk 1]
```diff
@@ -13,7 +13,7 @@
   <li>edit locations and vacancies for current courses</li>
   <% if @provider.accredited_body? %>
     <li>see which courses you're currently the accredited body for</li>
-    <li>request PE courses for the next cycle (and view requests you've already made)</li>
+    <li>view requests you've already made to recruit for fee-funded PE in the next cycle</li>
```
Comment (dankmitchell) [inline app/views/providers/_show_during_rollover.html.erb:16]:
```suggestion
    <li>view requests you’ve already made to recruit for fee-funded PE in the next cycle</li>
```

File: app/views/providers/_show_during_rollover.html.erb [hunk 2]
```diff
@@ -35,5 +35,6 @@
   <li>manage locations for the next cycle</li>
   <% if @provider.accredited_body? %>
     <li>see which courses in the next cycle you're the accredited body for</li>
+    <li>view requests you've already made to recruit for fee-funded PE in the next cycle</li>
```
Comment (dankmitchell) [inline app/views/providers/_show_during_rollover.html.erb:38]:
```suggestion
    <li>view requests you’ve already made to recruit for fee-funded PE in the next cycle</li>
```

**Review by dankmitchell — COMMENTED** [review]
Proper apostrophe

**Comment by AbigailMcP** [comment]
> Proper apostrophe

Thank you! Updated.

File: app/views/providers/_show_during_rollover.html.erb [hunk 3]
```diff
@@ -13,7 +13,7 @@
   <li>edit locations and vacancies for current courses</li>
   <% if @provider.accredited_body? %>
     <li>see which courses you're currently the accredited body for</li>
```
Comment (defong) [inline app/views/providers/_show_during_rollover.html.erb:15]:
```suggestion
    <li>see which courses you’re currently the accredited body for</li>
```

File: app/views/providers/_show_during_rollover.html.erb [hunk 4]
```diff
@@ -35,5 +35,6 @@
   <li>manage locations for the next cycle</li>
   <% if @provider.accredited_body? %>
     <li>see which courses in the next cycle you're the accredited body for</li>
```
Comment (defong) [inline app/views/providers/_show_during_rollover.html.erb:37]:
```suggestion
    <li>see which courses in the next cycle you’re the accredited body for</li>
```

**Review by defong — COMMENTED** [review]
It all starts with that smart quote. may as well be consistence and update the rest of the text....

**Review by defong — APPROVED** [review]

**Pull request merged**